docs/list.json

```diff
@@ -1,9 +1,7 @@
 {
-
+	
```

Comment:
It seems this file was unnecessarily formatted. Can you please revert these changes?

Reply:
I think this is resolved!